src/microservices/Microservices.DicomAnonymiser/Anonymisers/AnonymiserType.cs

```diff
@@ -6,5 +6,6 @@ public enum AnonymiserType
         /// Unused placeholder value
         /// </summary>
         None = 0,
+        DicomAnonymiser = 1,
```

Comment:
This name is too general since any `AnonymiserType` could be called `DicomAnonymiser`! I'm open to suggestions, but maybe `DefaultAnonymiser` or something that captures the actual implementation?

Reply:
That's a good suggestion there: `DefaultAnonymiser` captures the actual implementation in a better way.